app/workers/dependency_resolution_worker.rb

```diff
@@ -8,17 +8,15 @@ def perform(args = {})
     assign_attributes(args.deep_symbolize_keys)
 
     dependencies.each do |(content_id, locale)|
-      if draft?
-        downstream_draft(content_id, locale)
-      else
-        downstream_live(content_id, locale)
-      end
+      send_downstream(content_id, locale)
     end
+
+    orphaned_content_ids.each { |content_id| send_downstream(content_id, "en") }
```

Comment:
I think we'd want all locales to be included in this? (If I remember right that's what content dependencies does)